src/AssetsPage/index.jsx

```diff
@@ -17,11 +20,31 @@ class AssetsPage extends React.Component {
       });
   }
 
-  render() {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: 'off',
+    };
     AssetsPage.request();
+  }
+
+  handleCheckBoxChange = (event) => {
+    console.log(`checked: ${event}`);
+    this.setState({
+      checked: event,
+    });
```

Comment:
The checkbox is kind of broken right now so this doesn't really do anything. See https://github.com/edx/paragon/pull/26